CODE_OF_CONDUCT.md

```diff
@@ -130,11 +128,22 @@ You may also follow [NumFOCUS’s Code of Conduct violation reporting][NF-conduc
 guidelines if you would like to contact a third-party outside of the MDAnalysis 
 community.
 
+### Reporting at MDAnalysis Events and Meetups
+If you are attending an MDAnalysis event or meetup and wish to make a report, you may contact the ombudspersons (who will identify themselves at the event) or other event staff/meetup organizers so that they can take any appropriate immediate response. If you would prefer not to do that, please [submit a report][conduct-form] to MDAnalysis. 
+
+Ombudspeople and event staff/meetup organizers will be happy to help participants contact venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the event/meetup.
+
 ## Enforcement
 
-When a report is sent to us we will reply as soon as possible to confirm receipt;
-we strive to answer in less than 24 hours. We will review the incident and
-determine, to the best of our ability
+All reports of breaches of the code of conduct will be investigated and handled by the **MDAnalysis Code of Conduct Committee**. The current MDAnalysis Code of Conduct Committee consists of:
+- Jenna Swarthout Goddard (@jennaswa)
+- Micaela Matta (@micaela-matta)
+- Richard Gowers (@richardjgowers)
+In the event of any conflict of interest, the affected committee member will immediately notify the rest of the committee and recuse themselves if necessary.
```

Comment:
Remove the "if necessary" as it unnecessarily qualifies the initial strong statement. I believe that *if there's a potential conflict of interest* (real or perceived doesn't matter) *then they are just excusing themselves as a matter of policy*.
```suggestion
In the event of any conflict of interest, the affected committee member will immediately notify the rest of the committee and recuse themselves.
```